### ISSUE TITLE
OpenShot shortens clip instead of speeding it up

### ISSUE THREAD

**Issue by skuroedov** [OpenShot/openshot-qt#3081]
When I try to speed up clip, OpenShot shortens it instead of speeding it up.

How I do that:
1. Right click on clip
2. Time
3. Fast
4. Forward
5. 2x

**Expected behavior:**
Clip would be 2x faster

**System Details:**
- OpenShot Version: 2.4.3
- Operating System / Distro: ArchLinux GNU/Linux 5.3.7.

**Comment by speedytechdev** [comment]
Hello @skuroedov, your bug report has been tagged as "**Needs Info**" meaning we do not have the required info to test and solve the issue. It looks like you haven't attached your log files. If you're not sure how to get them please look at the image below. Please let us know if you still need help with getting the right info. 

Have a great day 
Speedytechdev

![image](https://user-images.githubusercontent.com/40452959/69215146-be78a400-0bcd-11ea-9f3f-6da782f2218a.png)

**Comment by ferdnyc** [comment]
Oh. Well, this is strange. I _thought_ I'd left a comment here, in addition to tagging as 'Needs info'. But I guess maybe I didn't submit the comment. Anyway, it was something like:

@skuroedov 
At the risk of stating the obvious, a clip that's played twice as fast _would_ be shorter, no? If it plays twice as fast, it will take half the time to play. So, OpenShot isn't shortening your clip "instead of" changing the speed, the shortening is a normal and expected effect of the speed-change process.

But are you saying that, when you speed up your clip, OpenShot changes the duration but it **doesn't** play back faster? So, is it only playing the first half of the clip, then?

The other thing is that 2.4.3 is very out of date, at this point. The issue you're having may be fixed in the newer release. Have you thought about upgrading to OpenShot 2.4.4, or even a more recent Daily Build version? You can run either directly from an AppImage downloaded at https://openshot.org/download/

**Comment by ferdnyc** [comment]
@speedytechdev Hm... is that some sort of auto-message for "need info"-tagged issues? That's a cool idea, more guidance for issue lifetime is definitely needed.

I'm just not sure embedding info as an image makes sense. It's a lot harder to read than it would be if the same info were just presented as text. GitHub is shrinking the image to fit the comment width, so the rendered font becomes really hard to make out. Also, it shows Windows paths (only), which aren't really useful on a Linux bug report like this one.

**Comment by speedytechdev** [comment]
@ferdnyc No it's not an automated message but it does sound like one. It would be cool if we could implement it somehow. I will make sure to use text next time.

**Comment by skuroedov** [comment]
@ferdnyc Yes, when I speed up a clip 2x, it should be 2x shorter. But the speed of the clip is normal and the clip just ends in half of itself.

**Comment by speedytechdev** [comment]
@skuroedov Have you tried the [daily build](https://www.openshot.org/download/)? I have been doing a lot of testing and I believe that issue was solved in there.

**Comment by skuroedov** [comment]
I tried now and the bug is not there anymore... But I tried other projects where was the behavior normal... Probably somehow corrupted project or something

**Comment by speedytechdev** [comment]
All good, If you can provide the save file and logs @ferdnyc can probably work out what happened to that project. Thanks for reporting the bug enjoy the rest of your day :)

**Comment by ferdnyc** [comment]
> All good, If you can provide the save file and logs @ferdnyc can probably work out what happened to that project.

_Possibly_, but honestly the Time (speed-change) property is by far the most fragile and unpredictable of the bunch (by like several orders of magnitude), so it never really surprises me when something breaks with it... all that's required is for someone to look at it funny. Deleting the clip and adding a new one to the timeline (IOW, clearing out all adjustments and starting with a clean import) usually clears up the problem, unless of course the same conditions cause it to occur in the new clip as well.

Long story short, point is that there are probably 100-200 GitHub issues here that all basically amount to: "Tried to use Time parameter to change playback speed of clip. It broke everything."  So... misery loves company, maybe?  `¯\_(ツ)_/¯`
